test/e2e/network/no_snat.go

```diff
@@ -52,70 +59,176 @@ var (
 				{
 					Name:  noSNATTestName,
 					Image: imageutils.GetE2EImage(imageutils.Agnhost),
-					Args:  []string{"netexec", "--http-port", testPodPort},
+					Args:  []string{"netexec", "--http-port", port},
 				},
 			},
+			HostNetwork: hostNetwork,
 		},
 	}
-)
+}
 
 // This test verifies that a Pod on each node in a cluster can talk to Pods on every other node without SNAT.
```

Comment:
It doesn't test _every_ node. "Pods on various nodes in a cluster can talk to Pods on various other nodes" ?